src/components/Modals/QrCode/Form.tsx

```diff
@@ -0,0 +1,154 @@
+import type { AccountId, AssetId, ChainId } from '@shapeshiftoss/caip'
+import { ethAssetId } from '@shapeshiftoss/caip'
+import type { FeeDataEstimate } from '@shapeshiftoss/chain-adapters'
+import { FeeDataKey } from '@shapeshiftoss/chain-adapters'
+import { AnimatePresence } from 'framer-motion'
+import { useCallback, useEffect } from 'react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { Redirect, Route, Switch, useHistory, useLocation } from 'react-router-dom'
+import { QrCodeScanner } from 'components/QrCodeScanner/QrCodeScanner'
+import { SelectAssetRouter } from 'components/SelectAssets/SelectAssetRouter'
+import { parseMaybeUrl } from 'lib/address/address'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+import {
+  selectAssetById,
+  selectMarketDataById,
+  selectSelectedCurrency,
+} from 'state/slices/selectors'
+import { store, useAppSelector } from 'state/store'
+
+import { useFormSend } from '../Send/hooks/useFormSend/useFormSend'
+import { SendFormFields, SendRoutes } from '../Send/SendCommon'
+import { Address } from '../Send/views/Address'
+import { Confirm } from '../Send/views/Confirm'
+import { Details } from '../Send/views/Details'
+
+export type SendInput<T extends ChainId = ChainId> = {
+  [SendFormFields.AccountId]: AccountId
+  [SendFormFields.To]: string
+  [SendFormFields.From]: string
+  [SendFormFields.AmountFieldError]: string | [string, { asset: string }]
+  [SendFormFields.AssetId]: AssetId
+  [SendFormFields.CryptoAmount]: string
+  [SendFormFields.EstimatedFees]: FeeDataEstimate<T>
+  [SendFormFields.FeeType]: FeeDataKey
+  [SendFormFields.FiatAmount]: string
+  [SendFormFields.FiatSymbol]: string
+  [SendFormFields.Input]: string
+  [SendFormFields.Memo]?: string
+  [SendFormFields.SendMax]: boolean
+  [SendFormFields.VanityAddress]: string
+}
```

Comment:
We've already defined this in `src/components/Modals/Send/Form.tsx`. Can we extract it and share it across both?

Reply:
@gomesalexandre please follow-up post-merge.

Reply:
Nice catch ser! https://github.com/shapeshift/web/pull/4418